### PR TITLE
User Form Project

### PR DESCRIPTION
**Pull request by emiliocm9**
## User Form for Microverse Curriculum

- In this project we developed a RoR application which manages a database to create users with information as username, password and email.
- This application is deployed on your local server.
- This application has the next features :
  - [x] You can create user containing its information as username, email and password.
  - [x] You can delete any user previously created.
  - [x] You can edit the information of any user previously created.
  - [x] There's an index page which shows all the users in the database.

### CODE REVIEW

File: app/views/users/new.html.erb [hunk 1]
```diff
@@ -0,0 +1,33 @@
+<%# <form accept-charset="UTF-8" action="new" method="get">
```
Comment (euqueme) [inline app/views/users/new.html.erb]: - [x]  uses users_path helper for action
- [x] uses `post` for method

File: app/views/users/new.html.erb [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  <input type="hidden" name="authenticity_token" value="<%= form_authenticity_token
+  <label for="username">User_name:</label>
+  <input type="text" id="username" name="username"><br><br>
```
Comment (euqueme) [inline app/views/users/new.html.erb]: - [x] uses `user[<field_name>]` for input's name

File: app/views/users/new.html.erb [hunk 3]
```diff
@@ -0,0 +1,33 @@
+<%# <form accept-charset="UTF-8" action="new" method="get">
+  <input type="hidden" name="authenticity_token" value="<%= form_authenticity_token
```
Comment (euqueme) [inline app/views/users/new.html.erb]: - [x] this line is missing closing brackets and quotation let's fix it

**Review by euqueme — CHANGES_REQUESTED** [review]
> ### Pull Request reviewed by Maria Eugenia Queme P. (TSE)

### Hello @emiliocm9 and @brugobi :lollipop: :balloon: 

### Thank you for your submission :handshake: :handshake:  your project works great and I like you worried about displaying validation errors in the views and that you deploy it in heroku :medal_sports: :medal_sports:  there are a few changes to make so let's work on them :construction: :construction_worker_man: :construction_worker_woman: 

#### Form format

- [x] I've left comments in your code where changes have to be made to the commented forms.

##### [OPTIONAL]

- [x] Create a partial for the user form and reuse it for the edit and new views

### That's it, team! :raised_hands: 

### Please :pray:  do the changes :technologist: :woman_technologist:  and submit back for a code review when you are done :memo: 

Happy coding! :rocket: 

Any question? ask me on slack **@ Maru** :woman_technologist: 

PROJECT STATUS: REQUESTED CHANGES :vertical_traffic_light:

**Comment by Anna-Myzukina** [comment]
# [APPROVED]  :heavy_check_mark: 

## Thanks for applying the changes  :+1:

### Go and merge to development.

# :raising_hand:  See you at the next projects.

Happy coding  <g-emoji class="g-emoji" alias="computer" fallback-src="https://github.githubassets.com/images/icons/emoji/unicode/1f4bb.png">computer</g-emoji>

**Pull request merged**